technical-reports/resolver/terminology.md

```diff
@@ -1,103 +1,37 @@
-# Resolver Terminology
+# Terminology
 
-<dl>
-<dt>Resolver</dt>
-<dd>
+## <dfn>Resolver</dfn>
```

Comment:
Swapped from a definition list, which is NOT bookmarkable, to a simple heading hierarchy which is.

In ReSpec, adding `<dfn>…</dfn>` around anything allows it to be accessed with `[=someterm=]` throughout the document which is vital for reuse. We’ll make heavy use of this.